src/idl_parser.cpp

```diff
@@ -191,7 +191,7 @@ std::string Parser::TokenToStringId(int t) {
 // Parses exactly nibbles worth of hex digits into a number, or error.
 CheckedError Parser::ParseHexNum(int nibbles, int64_t *val) {
   for (int i = 0; i < nibbles; i++)
-    if (!isxdigit(cursor_[i]))
+    if (!isxdigit(static_cast<const unsigned char>(cursor_[i])))
       return Error("escape code must be followed by " + NumToString(nibbles) +
```

Comment:
isxdigit takes an int, and we supply a char. what warning exactly is this giving?